### ISSUE TITLE
Enable empty-heading rule

### ISSUE THREAD

**Issue by v-viyada** [microsoft/accessibility-insights-web#7423]
## Is your feature request related to a problem? Please describe.
I am running into an issue on the url [Offer Benefits | Microsoft ISV Hub](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.microsoft.com%2Fen-us%2Fisv%2Foffer-benefits&data=05%7C02%7Cv-viyada%40microsoft.com%7C75acd231c94e410cef5108dcbe368c90%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638594388707710295%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=p7xmacFioGlEM90ecLWtc3paumLIIb4%2FZEm23Yoc%2FVc%3D&reserved=0) about the “two empty headings” that was caught by “Wave” accessibility tool but not with the “AI” fast pass.

## Describe the desired outcome
AI fast pass should catch empty-heading failure instances.

## Additional context
Details for internal thread:
A11y Team: As per axe-core rule documentation https://dequeuniversity.com/rules/axe/4.9, empty-heading is best practice rule and do not necessarily conform to WCAG success criterion. More details on axe-core rule: https://dequeuniversity.com/rules/axe/4.9/empty-heading. Please raise a github issue and we will discuss in triage meeting.

User Response: 
Solving for an empty heading issue isn't just best practice; it's a WCAG violation under SC 2.4.6, [Understanding Success Criterion 2.4.6: Headings and Labels | WAI | W3C](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FWAI%2FWCAG21%2FUnderstanding%2Fheadings-and-labels.html&data=05%7C02%7Cv-viyada%40microsoft.com%7C75acd231c94e410cef5108dcbe368c90%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638594388707654817%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=%2BUFCSU8md0wPepyWiqrvCDfSuxCccKGUaWIBwED7HKo%3D&reserved=0)  which requires headings to be descriptive. An empty heading can't meet this requirement, leading to confusion and disrupting the page's structure. This is particularly problematic for low vision users who depend on screen readers or partial screen views.

**Comment by scottaohara** [comment]
is this issue is flagged, it shouldn't be under 2.4.6 - 1.3.1 would be more appropriate.

**Comment by joestegman425** [comment]
After further investigation, we have decided not to enable this rule due to low user impact. The reason this is low impact is that most ATs ignore empty headings and in cases where they are not ignored, the user impact is limited.
